Documentation/ConfigFile.md

```diff
@@ -132,13 +132,26 @@ If you need more control when using a configuration file, you can use some advan
   swiftgen config init --config tools/swiftgen/swiftgen-config.yml
   ```
   
-* Enable the verbose mode, which will print every command being executed when executing it, using the `--verbose` flag. This allows your to:
```

Comment:
Obviously tweak docs regarding the `--quiet` flag

Reply:
Made the changes